R/testGeno.R

```diff
@@ -34,36 +34,36 @@ testGenoSingleVar <- function(nullmod, G, E = NULL, test = c("Score", "Score.SPA
     if(calc.score){
         Gtilde <- calcGtilde(nullmod, G)
         if(is.null(nullmod$RSS0)){
```

Comment:
@amstilp this RSS0 calculation could actually move into your "update old null models" function I think. This is also a fix for old versions.

Reply:
Good point -- done.